src/fluree/server/task/migrate_sid.clj

```diff
@@ -0,0 +1,52 @@
+(ns fluree.server.task.migrate-sid
+  (:require [clojure.core.async :as async :refer [<! >! go-loop]]
+            [fluree.db.json-ld.migrate.sid :as migrate-sid]
+            [fluree.db.util.async :refer [go-try]]
+            [fluree.db.util.core :as util]
+            [fluree.db.util.log :as log]
+            [malli.core :as m]))
+
+(def MigrateSidTask
```

Comment:
I think we should do all validation up front, and not scatter malli throughout the codebase. This schema should live in the `fluree.server.config` namespace.